src/include/simeng/OS/Constants.hh

```diff
@@ -0,0 +1,100 @@
+#pragma once
+#include <stdint.h>
+
+namespace simeng {
+namespace OS {
+
+namespace defaults {
+
+/** The Default PageSize for SimEng. */
+static constexpr uint64_t page_size = 4096;
+
+/** The maximum number of file descriptors a process can have,as defined by the
+ * linux kernel.*/
+static constexpr uint64_t maxFdNum = 1024;
+
+}  // namespace defaults
+
+/** These syscall constants are mainly used in MemRegion to ensure we are
+ * consistent will linux specific definitions in different Operating systems. */
```

Comment:
I think slightly different wording would make it clearer : "The syscall constants are mainly used in MemRegion to ensure we are consistently using the Linux specific definitions of certain syscall arguments, regardless of the host Operating System SimEng is being run on."

Reply:
done